### PR TITLE
Features: Add binding to get the creation timestamp of an image or a …

### PR DESCRIPTION
**Pull request by fflorens**
…snapshot

    Add the following functions:
    func (image *Image) GetCreationTime() (timestamp time.Time, err error)
    func (image *Image) GetSnapCreationTime(snap_name string) (timestamp time.Time, err error)

    and also a helper function:
    func (image *Image) GetSnapId(snap_name string) (snapId uint64, err error)

I am currently writing a grpc/http api that needs to have access to those, and i though i would share :)

### CODE REVIEW

File: rbd/rbd.go [hunk 1]
```diff
@@ -809,6 +823,36 @@ func (image *Image) GetSnapshotNames() (snaps []SnapInfo, err error) {
...
+// int rbd_snap_get_timestamp(rbd_image_t image, uint64_t snap_id, struct timespec *timestamp);
+func (image *Image) GetSnapCreationTime(snap_name string) (timestamp time.Time, err error) {
+  var c_timespec C.struct_timespec
+
+  snapId, err := image.GetSnapId(snap_name)
+  if err != nil {
+    return time.Unix(0,0), err
+  }
+
+  ret := C.rbd_snap_get_timestamp(image.image, C.uint64_t(snapId), &c_timespec); ret < 0 {
```
Comment (dswarbrick) [inline rbd/rbd.go]: This appears to be missing an "if" at the start of the line, and a closing curly bracket for the block. Please run `go fmt` on your code - you'll save yourself the hassle of a rejected PR when the CI bot runs.

Comment (fflorens) [reply rbd/rbd.go]: Updated, sorry for the missing statements ;)

**Comment by noahdesu** [comment]
@fflorens could you please add a unit test?

**Comment by fflorens** [comment]
I've added unit tests,
At the same time I modified the name of the function to keep consistency in the name.

Also added checks for the case where the image is not opened

**Comment by fflorens** [comment]
Any news ?

**Comment by albttx** [comment]
Up ? need too !

**Comment by phlogistonjohn** [comment]
@fflorens  Hi, thank you for your contributions to this project. This project had gone stale for a while and some of us are new contributors trying to update it and actively contribute. However, because we are not always familiar with the context of changes that were started a while ago before we were involved I'd like to ask you to update this PR with your current status and if you feel you would still like us to review it anew. If we do not hear from you in a few weeks we will probably close the PR as stale in order to lower our backlog. Thanks again!

**Comment by phlogistonjohn** [comment]
Closing PR after no author response.

**Pull request closed without merging**